l10n_ch_fds_postfinance/models/fds_postfinance_file.py

```diff
@@ -106,15 +106,18 @@ def import2bankStatements(self):
         res = True
         for pf_file in self:
             try:
-                values = {'data_file': pf_file.data}
+                values = {
+                    'data_file': pf_file.data,
+                    'filename': pf_file.filename
+                }
                 bs_import_obj = self.env['account.bank.statement.import']
                 bank_wiz_imp = bs_import_obj.create(values)
                 import_result = bank_wiz_imp.import_file()
                 # Mark the file as imported, remove binary as it should be
                 # attached to the statement.
                 pf_file.write({
                     'state': 'done',
-                    'data': None,
+                    'data': pf_file.data,
```

Comment:
You can simply remove this line instead. pf_file does already contain the data, so this line is doing nothing.